Library/Homebrew/os/mac.rb

```diff
@@ -204,7 +204,11 @@ def preferred_arch
       "10.1"   => { clang: "10.0", clang_build: 1000 },
       "10.2"   => { clang: "10.0", clang_build: 1001 },
       "10.2.1" => { clang: "10.0", clang_build: 1001 },
+      "10.3"   => { clang: "10.0", clang_build: 1001 },
       "11.0"   => { clang: "11.0", clang_build: 1100 },
+      "11.1"   => { clang: "11.0", clang_build: 1100 },
+      "11.2"   => { clang: "11.0", clang_build: 1100 },
+      "11.2.1" => { clang: "11.0", clang_build: 1100 },
     }.freeze
 
     def compilers_standard?
```

Comment:
There don't seem to have been any issues reported from having missed updating this, and I can't find where `compilers_standard?` is used – maybe this isn't necessary anymore? Opened the PR for discussion.

Reply:
Yeah seems the last ref was deleted some years ago now: https://github.com/Homebrew/brew/commit/77c0f60434aa9c51ef5b052a43a200ea2b39f04f

Reply:
Opened #6787 as an alternative.